.github/ISSUE_TEMPLATE/give_a_talk.yml

```
@@ -0,0 +1,43 @@
+name: Give a TTW talk 
+description: Resources and checklist for giving a talk about the Turing Way
+title: "[TALK] <title>"
+labels: [talks-and-workshops, newsletter]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This template will guide you through the steps necessary to give a successful talk about The Turing Way!
+  - type: input
+    id: date
+    attributes:
+      label: Date of talk
+      description: When will you be presenting about the Turing Way?
+      placeholder: please use the format YYYY-MM-DD
+    validations:
+      required: true
+  - type: input
```

```suggestion
  - type: textarea
```

I would make this a larger text area so users can provide more info if they would like. It's just a bit more freeform than the `input` type.